autoflake.py

```diff
@@ -1053,12 +1058,31 @@ def _main(argv, standard_out, standard_error):
 
     filenames = list(set(args.files))
     failure = False
-    for name in find_files(filenames, args.recursive, args.exclude):
-        try:
-            fix_file(name, args=args, standard_out=standard_out)
-        except IOError as exception:
-            _LOGGER.error(unicode(exception))
-            failure = True
+
+    # convert argparse namespace to a dict
+    args = vars(args)
```

Comment:
Any reason for using a dict instead of the regular namespace? Is it for serialization?